jsonpipe.js

```diff
@@ -99,6 +99,31 @@ module.exports = {
 },{"./net/xhr":2}],2:[function(_dereq_,module,exports){
 'use strict';
 
+var trim = ''.trim
+  ? function(s) { return s.trim(); }
+  : function(s) { return s.replace(/(^\s*|\s*$)/g, ''); }
+
+function parseHeader(str) {
+  var lines = str.split(/\r?\n/);
+  var fields = {};
+  var index;
+  var line;
+  var field;
+  var val;
+
+  lines.pop(); // trailing CRLF
+
+  for (var i = 0, len = lines.length; i < len; ++i) {
+    line = lines[i];
+    index = line.indexOf(':');
+    field = line.slice(0, index).toLowerCase();
```

Comment:
@yaslama is the `toLowerCase()` needed? Can we just have it as the way it comes from the response?